src/views/parents/parents.jsx

```diff
@@ -0,0 +1,128 @@
+                                <p><FormattedMessage id='parents.privacyPolicyDescription' /></p>
+                            </li>
+
+                            <li>
+                                <h3><FormattedMessage id='parents.noOnline' /></h3>
+                                <p><FormattedMessage id='parents.noOnlineDescription' /></p>
+                            </li>
+
+                            <li>
+                                <h3><FormattedMessage id='parents.parentsSaying' /></h3>
+                                <p><FormattedMessage id='parents.parentsSayingDescription' /></p>
+                                <p><b>"I just want to thank you all for making Scratch, and{' '}
+                                   for providing it for free.</b> My kids are doing amazing{' '}
+                                   things that they see as fun yet I know is educational,{' '}
+                                   valuable, and worthwhile. Thank you so much!!!"</p>
+                                <p><b>"My very shy but technical minded daughter has found{' '}
+                                   this to be a fantastic, safeoutlet for her creativity.</b>{' '}
+                                   She spends her free time creating ever more difficult{' '}
+                                   animations and sharing them with the scratch community.{' '}
+                                   The forums provide her with a group of like-minded{' '}
+                                   individuals with which she can hold on a conversation...{' '}
+                                   She now feels that computers, graphic design and{' '}
+                                   animation are something she would like to pursue in the{' '}
+                                   future. Your program has opened a whole new world to her{' '}
+                                   in so many ways, and I thank you wholeheartedly. "</p>
+                                <p><b>"My son is learning more than I can imagine from your{' '}
+                                   tool.</b> He is not a natural logical thinker but loves{' '}
+                                   LEGO. Your LEGO-like building block structure has moved him{' '}
+                                   forward by light-years in his logical thinking skills...He{' '}
+                                   can snap things together and begin to see the logic{' '}
+                                   reinforced by immediate feedback. Of course we work on{' '}
+                                   some things together--instant father and son time. This{' '}
+                                   is just incredible. Just a big thanks to you and MIT."</p>
+                            </li>
+                        </ul>
+                    </FlexRow>
+                </div>
+                
+                <div className="bodySide">
+                    <FlexRow>
+                        <Box title={''}>
```

Comment:
There seem to be a few built in `Box` stylings that are giving it a default width wider than we want. I'd try to solve this using the `FlexRow with uneven` changes I proposed above, but if that doesn't work let me know! Here's a screen shot of what it looks like now:
<img width="1036" alt="screen shot 2016-10-17 at 3 34 57 pm" src="https://cloud.githubusercontent.com/assets/1815772/19453031/733cd720-9481-11e6-89aa-2736e667c41d.png">